frontend/.babelrc.js

```diff
@@ -0,0 +1,6 @@
+const plugins = ["lodash"];
+
+module.exports = {
+  plugins,
+  presets: ["next/babel"],
+};
```

Comment:
Nit: Nice! For my own understanding, what was the benefit of migrating to a `.babelrc.js`?

Reply:
https://babeljs.io/docs/en/configuration#javascript-configuration-files
It extends the json one. Not a huge benefite but as far as I know, it's the more standardized way :-)